app/assets/stylesheets/concepts.scss

```diff
@@ -167,3 +167,9 @@ div.synonym-change-request button {
   margin-top: -4px;
 }
 
+#details_content .chip-button-component-container {
```

Comment:
why not generalizing this to everything, because this happen also in schemes or collections tabs too